prep-html-syntax/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <h1 style="color: green;">My favorite Star Wars character is Yoda!
+    </h1>
+    <img src="https://img.money.com/2019/11/baby-yoda-toys-for-sale-1163281433.jpg?quality=60&w=800" alt="Yoda"
+  <br>
+    <p style="color: rgb(104, 68, 22);">Yoda may look frail but he is wise and powerful!
+    </p>
+  </body>
```

Comment:
Hi Simon, be sure to divide your page into two specific sections. The `<head>` should open and close before the `<body>` opens and closes. The visible content of the page should go in the `<body>`.

```html
<!DOCTYPE html>
<html>
  <head>
    <!-- meta info like <title></title> goes in here -->
  </head>
  <body>
    <!-- visible content like <h1></h1> and <p></p> goes in here -->
  </body>
</html>
```

Reply:
Hi @Skim-329 I was just referring to the missing `<head>` element. Looks good now!